test/segment-loader.test.js

```diff
@@ -249,7 +249,6 @@ QUnit.test('calculates bandwidth after downloading a segment', function(assert)
   // verify stats
   assert.equal(loader.mediaBytesTransferred, 10, '10 bytes');
   assert.equal(loader.mediaTransferDuration, 100, '100 ms (clock above)');
-  assert.equal(loader.mediaRequests, 1, '1 request');
```

Comment:
Because we have this same test dozens of times and it's very redundant. I actually tried to remove more of them but lost hope.